modules/ebi-metagenomics/keggpathwayscompleteness/main.nf

```diff
@@ -12,14 +12,14 @@ process KEGGPATHWAYSCOMPLETENESS {
     tuple val(meta), path(ko_contig_tsv)
 
     output:
-    tuple val(meta), path("${prefix}_summary_kegg_pathways.tsv"), emit: kegg_pathways
-    path "versions.yml",                                          emit: versions
+    tuple val(meta), path("${prefix}_summary_kegg_pathways.tsv"),             emit: kegg_pathways
```

Comment:
if you add `--outprefix` arg you can catch files like
`path("${prefix}/${prefix}_summary_kegg_pathways.tsv")` and not need to remane those inside script
or even you can remove -o and it should produce output into current directory

Reply:
https://www.ebi.ac.uk/panda/jira/browse/EMG-7787